plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -963,6 +969,7 @@ public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle
                     handle.getMetadataEntry().getId(),
                     columnsBuilder.build(),
                     partitionColumns,
+                    ImmutableMap.of(),
```

Comment:
Can you add a test that creates a table with comments and then adds a column? Make sure this doesn't wipe the existing comments